File: app/src/main/java/com/example/myapplication/OnboardingActivity.java
package com.example.myapplication;

import android.content.Intent;
import android.os.Bundle;
import android.view.View;
import android.widget.TextView;

import androidx.appcompat.app.AppCompatActivity;

public class OnboardingActivity extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.onboard);

        // Initialize TextView
        TextView loginText = findViewById(R.id.textView3);

        // Set Click Listener for "Already have an account? Login"
        loginText.setOnClickListener(v -> {
            Intent intent = new Intent(OnboardingActivity.this, LoginActivity.class);
            startActivity(intent);
        });
    }
}

File: app/src/main/java/com/example/myapplication/OnboardingActivity.kt
package com.example.myapplication

import android.os.Build
import android.os.Bundle
import androidx.activity.ComponentActivity
import androidx.activity.compose.setContent
import androidx.annotation.RequiresApi
import androidx.lifecycle.viewmodel.compose.viewModel
import com.example.myapplication.ui.theme.BetterAppTheme
import com.example.myapplication.navigation.AppNavigation
import com.example.myapplication.viewmodel.FallDetectionViewModel
import com.example.myapplication.viewmodel.MainViewModel

class OnboardingActivity : ComponentActivity() {
    @RequiresApi(Build.VERSION_CODES.O)
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContent {
            val mainViewModel: MainViewModel = viewModel()
            val fallDetectionViewModel: FallDetectionViewModel = viewModel()

            BetterAppTheme {
                AppNavigation(mainViewModel, fallDetectionViewModel)
            }
        }
    }
}
